src/server/routes/merge.ts

```diff
@@ -274,6 +305,14 @@ router.get('/add/:bbid', auth.isAuthenticated,
 		if (type !== mergeQueue.entityType) {
 			mergeQueue.mergingEntities = {};
 			mergeQueue.entityType = _.upperFirst(type);
+			mergeQueue.seriesType = type === 'Series' ? fetchedEntity.entityType : null;
```

Comment:
This would benefit from having a comment; at first I didn't understand the distinction between entityType in the context of the mergeQueue and entityType in the context of a Series entity